CHANGES.md

```diff
@@ -36,6 +36,8 @@ To be released.
         Instead, a newly added static class `StoreExtension` provides
         an extension method of the same name.
      -  Added `IStore.IterateStateReferences()` method.
+ -  `Swarm` constructor became to receive `BlockChain<T>`.  [[#324]]
+ -  `Swarm` became to have type parameter `T`.  [[#324]]
```

Comment:
it would be better if there is mention about `T`.

```suggestion
 -  `Swarm` became to have type parameter `T` that reprsents action type as like as `BlockChain<T>`.  [[#324]]
```